Whiskey/Whiskey.psd1

```diff
@@ -150,7 +150,7 @@
 # 0.35.0
 
 * Created "SetVariableFromPowerShellDataFile" task for creating variables from values in PowerShell data files (e.g. .psd1 files, module manifests, etc.).
-* Fixed: `ProGetUniversalPackage` task should remove the temporary robocopy output file after it's no longer necessary so it doesn't inadvertently get added to package with a "*.txt" wildcard Include item.
+* Fixed: `ProGetUniversalPackage` task now uses a separate package staging directory inside the root task temp directory to prevent any sort of miscellaneous task files from being included in the generated package.

```

Comment:
I would rephrase this as "`ProGetUniversalPackage` task no longer includes robocopy output text file in your package."